engine/src/main/java/io/zeebe/engine/processing/bpmn/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -120,6 +125,16 @@ public BpmnElementContext transitionToCompleting(final BpmnElementContext contex
     return transitionedContext;
   }
 
+  private void verifyIncidentResolving(final BpmnElementContext context, final String methodName) {
```

Comment:
Please add a TODO pointing to #6202 so we can easily find this at the end.

Reply:
Also it's not obvious to me why we only want to throw this if not resolving an incident?

Reply:
On incidents we call the processing of the processor again, would this help as a comment or what would help here?

Reply:
Ah, I see, that makes sense. Yeah that helps for anyone who hasn't dealt with incidents in a while :slightly_smiling_face: I'm on the fence about it as it's just knowledge of the engine though...I guess it threw me off because it inspects the stack trace :sweat_smile: This is only temporary anyway right?

Reply:
Yeah I hope I can remove it as soon as possible